gh_actions_exporter/main.py

```diff
@@ -35,13 +41,15 @@ async def webhook(
     webhook: WebHook,
     request: Request,
     settings: Settings = Depends(get_settings),
-    metrics: Metrics = Depends(metrics)
+    metrics: Metrics = Depends(metrics),
+    cost: Cost = Depends(cost),
```

Comment:
I don't understand why cost is set as fastapi dependency. Because it feels to me that it can be called like a normal function from anywhere. (I could be wrong though).

Reply:
You're right, it's fixed